docs/stdlib/array.rst

```diff
@@ -130,9 +127,9 @@ Reference
 
 .. eql:operator:: arrayidx: array<anytype> [ int64 ] -> anytype
 
-    Array indexing.
+    Indexes an array of :eql:type:`anytype`.
```

Comment:
```suggestion
    Accesses the element of the array at a given index.
    
    Arrays in EdgeDB are zero-indexed.
```

Clarify explanation and demote zero-indexed to the description.

Reply:
Can we italicize "array" as what it's named in the argument signature or re-word it to use "an" instead? "the array" sounds a little jarring to me